config/locales/client.en.yml

```diff
@@ -1486,12 +1483,49 @@ en:
           error: "There was an error generating Invite link"
           max_redemptions_allowed_label: "How many people are allowed to register using this link?"
           expires_at: "When will this invite link expire?"
+
+        invite:
+          new_title: "Create Invite"
+          edit_title: "Edit Invite"
+          show_link: "Invite Link"
+
+          instructions: "Share this link to instantly grant access to this site:"
+          no_invite_link: "The invite link will show up here after created"
```

Comment:
```suggestion
          no_invite_link: "The invite link will show up here after it's created"
```

Reply:
Since the invite links are generated when the modal is displayed, this placeholder does not make sense anymore.